docs/source/common/trainer.rst

```diff
@@ -907,6 +907,9 @@ Stop training once this number of epochs is reached
     # default used by the Trainer
     trainer = Trainer(max_epochs=1000)
 
+If both ``max_epochs`` and ``max_steps`` aren't specified, ``max_epochs`` will default to ``1000``.
+To disable this default, you can set one (or both) of ``max_epochs`` or ``max_steps`` to be ``-1``.
+
```

Comment:
why are we mentioning both here? this is too complicated. imo it should simply say, to disable the limit for epochs, set max_epochs=-1.

Reply:
Done